package.json

```diff
@@ -65,6 +67,14 @@
     "eslint": "^3.19.0",
     "eslint-config-airbnb-base": "^11.1.3",
     "eslint-plugin-import": "^2.2.0",
+    "gulp": "^3.9.1",
```

Comment:
You probably need to rebase you branch. Gulp was already installed by @Mercy-Muchai